pydatastructs/miscellaneous_data_structures/queue.py

```diff
@@ -180,3 +179,56 @@ def __len__(self):
 
     def __str__(self):
         return str(self.queue)
+class PriorityQueue(Queue):
```

Comment:
is there a difference between the implementation in priority_queue.py and this one?